audio/Cargo.toml

```diff
@@ -14,12 +14,15 @@ lewton = "0.8.0"
 log = "0.3.5"
 num-bigint = "0.1.35"
 num-traits = "0.1.36"
-rust-crypto = { git = "https://github.com/awmath/rust-crypto.git", branch = "avx2" }
+rust-crypto = "0.2.36"
 tempfile = "2.1"
 
 tremor = { git = "https://github.com/plietar/rust-tremor", optional = true }
 vorbis = { version ="0.1.0", optional = true }
 
+[replace]
```

Comment:
These should only be at the root though.

Reply:
so [replace] propagates to dependencies?